aurora/third-parties/news-site-next-third-party-basic/.env.example

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_GOOGLE_API_KEY="ABC"
+NEXT_PUBLIC_GOOGLE_ANALYTICS_KEY="G-ABC"
+NEXT_PUBLIC_GOOGLE_TAG_MANAGER_KEY="GTM-ABC"
```

Comment:
These API keys aren't going to work and the 3Ps won't fully load correctly right? Or is the idea that these workloads don't necessarily need to as long as they fetch their scripts

Reply:
you'll need a .env file for it to work, with the correct values